phpBB/phpbb/cache/driver/file.php

```diff
@@ -584,4 +587,15 @@ function _write($filename, $data = null, $expires = 0, $query = '')
 
 		return $return_value;
 	}
+
+	/**
+	* Replace slashes in the file name
+	*
+	* @param string $name filename
+	* @return string the corrected name
+	*/
+	function replace_slashes($name)
```

Comment:
We should have the "protected" keyword before "function" here since this function is only used internally in the file class.